### PR TITLE
Dh home page UI

### PR DESCRIPTION
**Pull request by Dimo-Hristov**
Moved header component into core folder and make it global. Style home page based on design.

### CODE REVIEW

File: src/app/home/events-type-list/events-type-list.component.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
+<div class="upcoming-events-title">
+  <h2>Предстоящи събития</h2>
```
Comment (Getsov) [inline src/app/home/events-type-list/events-type-list.component.html]: Категории

File: src/app/home/home.page.ts [hunk 2]
```diff
@@ -15,50 +15,88 @@ export class HomePage implements OnInit {
     // TODO : fetch lastest events data
     this.upcomingEvents = [
```
Comment (Getsov) [inline src/app/home/home.page.ts:16]: use interface and use the real model

File: src/app/home/home.page.ts [hunk 3]
```diff
@@ -15,50 +15,88 @@ export class HomePage implements OnInit {
...
+      {
+        banner: '../../assets/icon/Image.jpg',
+        _id: 'asdasdasd',
+        shortTitle: 'Кондофрей Драг 2023',
+        startDate: '12-11-2023',
+        endDate: '14.11.23',
       },
     ];
     // TODO: fetch events type list
     this.eventsType = [
```
Comment (Getsov) [inline src/app/home/home.page.ts:68]: use Interface with the real model

File: src/app/home/events-type-list/events-type-list.component.html [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+  <div class="second-line"></div>
+</div>
+
 <ion-list lines="none" class="categories-list">
   <ion-card class="category-card all">
-    <ion-card-header class="category-title"> All </ion-card-header>
+    <ion-card-header class="category-title">Всички категории</ion-card-header>
     <img
-      src="../../../assets/icon/final-flag.jpg"
+      src="../../../assets//icon/category-icons/all.jpg"
```
Comment (Getsov) [inline src/app/home/events-type-list/events-type-list.component.html:11]: is the new route working?

Comment (Dimo-Hristov) [reply src/app/home/events-type-list/events-type-list.component.html:11]: yes

File: src/app/home/events-type-list/events-type-list.component.html (hunk 1 above)
Comment (Dimo-Hristov) [reply src/app/home/events-type-list/events-type-list.component.html]: Имаш око.

File: src/app/home/home.page.ts (hunk 2 above)
Comment (Dimo-Hristov) [reply src/app/home/home.page.ts:16]: didnt see interface yet

File: src/app/home/home.page.ts (hunk 3 above)
Comment (Dimo-Hristov) [reply src/app/home/home.page.ts:68]: didnt see interface yet

**Review by Getsov — APPROVED** [review]

**Pull request merged**